using-keycloak/src/main/resources/application.properties

```diff
@@ -5,6 +5,7 @@ quarkus.keycloak.auth-server-url=http://localhost:8180/auth
 quarkus.keycloak.resource=backend-service
 quarkus.keycloak.bearer-only=true
 quarkus.keycloak.credentials.secret=secret
+quarkus.keycloak.policy-enforcer.enable=true
```

Comment:
The title of your PR says it should be disabled but you enabled it?

BTW, is this something we want to disable by default in Quarkus itself?

Does the property just below make sense if you disable it?

Reply:
The fix is based on https://github.com/quarkusio/quarkus/issues/2422. For this quickstart, in particular, we are enabling the `keycloak.policy-enforcer` configuration (which should be optional).

Reply:
Could you make the commit message clearer then? thanks!

Reply:
@gsmet, I think the title now should be better ...